testsuite/features/secondary/allcli_action_chain.feature

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2018-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-@sle_client
+@centos_minion
```

Comment:
I wonder why we replace the SLE client with the CentOS minion in this feature. I guess because we already have a SLE minion and did the scenarios for the SLE client. I am wonderning if we need the cuplicate testing for a Salt minion here?

Reply:
Those features starting with allcli_ are features using multiple systems. What I did is to replace in some cases the sle_client by a centos_minion because both are using rpm files.
As the other option we have are the ubuntu_minion using deb files. And ofc, the sle_minion which is already in use in this feature.